packages/@sanity/desk-tool/src/pane/DocumentPane.js

```diff
@@ -41,6 +41,7 @@ import {historyIsEnabled} from './Editor/history'
 import menuItemStyles from './styles/documentPaneMenuItems.css'
 import {getDocumentPaneFooterActions} from './documentPaneFooterActions'
 import {getProductionPreviewItem, getMenuItems} from './documentPaneMenuItems'
+import {ClientError} from '@sanity/client'
```

Comment:
`@sanity/client` is not a (direct) dependency of `@sanity/desk-tool` - see below